README.md

```diff
@@ -33,33 +33,45 @@ The end result is that the remote cluster is seen by your IDE and tools as a loc
 - **Realistic development environment**: your development container reuses the same variables, secrets, sidecars, volumes, etc... than your original Kubernetes deployment. Realistic environments eliminate integration issues.
 - **Replicability**: development containers eliminate the need to install your dependencies locally, everything is pre-configured in your development image.
 - **Unlimited resources**: get access to the hardware and network of your cluster when developing your application.
-- **Deployment independent**: `okteto` decouples deployment from development. You can deploy your application with kubectl, Helm, a serverless framework, or even a CI pipeline and use `okteto up` to develop it. This is especially useful for cloud-native applications where deployment pipelines are not trivial.
-- **Works anywhere**: `okteto` works with any Kubernetes cluster, local or remote. `okteto` is also available for macOS, Linux, and Windows.
+- **Deployment independent**: the Okteto CLI decouples deployment from development. You can deploy your application with kubectl, Helm, a serverless framework, or even a CI pipeline and use `okteto up` to develop it. This is especially useful for cloud-native applications where deployment pipelines are not trivial.
+- **Works anywhere**: the Okteto CLI works with any Kubernetes cluster, local or remote. The Okteto CLI is also available for macOS, Linux, and Windows.
 
 ## Getting started
 
-All you need to get started is to [install the Okteto CLI](https://www.okteto.com/docs/getting-started/#installing-okteto-cli) and have access to a Kubernetes cluster.
+All you need to get started is:
 
-Okteto CLI works with **any** Kubernetes cluster. If it's your first time using it, we'd recommend you [try it](https://www.okteto.com/docs/getting-started/) with the [Okteto Platform](https://cloud.okteto.com/) for a complete holistic developer experience. If you want to try it out with any other K8s cluster, you can also check out [this article](https://www.okteto.com/blog/developing-microservices-by-hot-reloading-on-kubernetes-clusters/) as a guide.
+- Deploy your application on Kubernetes.
+- [Install the Okteto CLI](https://www.okteto.com/docs/getting-started/#installing-okteto-cli)
+- Configure the Okteto CLI to access to your Kubernetes cluster with `okteto context`
+- Run `okteto init` from the root of your application git repository to inspect your code and generate your [Okteto manifest](https://okteto.com/docs/reference/manifest/). The Okteto manifest defines your development container
+- Run `okteto up` to activate your development container
 
-We created a [few guides to help you get started](https://github.com/okteto/samples) with `okteto` and your favorite programming language.
+We created a few guides to help you get started with the Okteto CLI and your favorite programming languages:
```

Comment:
this requires more explanation on cli standalone vs cli + cluster modes, and why we have samples embedded here vs samples in okteto/docs and on okteto/samples.

I find that just having a list of samples here confusing.